load-test/bin/generate-pupil-data.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+'use strict'
+
+// require('dotenv').config()
+const winston = require('winston')
+const upnService = require('../../admin/services/upn.service')
+winston.level = 'info'
+const moment = require('moment')
+
+const poolService = require('../../admin/services/data-access/sql.pool.service')
+const sqlService = require('../../admin/services/data-access/sql.service')
+const pupilCountPerSchool = 40
```

Comment:
@jon-shipley would we ever want this to be dynamic coming as an arg?

Reply:
That's a great idea!  Feel free to mod the script.